core/controllers/base.py

```diff
@@ -330,6 +330,8 @@ def render_template(
                 self.username and self.username in
                 config_domain.WHITELISTED_COLLECTION_EDITOR_USERNAMES.value
             ),
+            'promo_bar_enabled': config_domain.PROMO_BAR_ENABLED.value,
```

Comment:
I suspect this promo bar is generally a good thing for us to have. I'll add a feconf variable to gate this, so in the very least we can do a trivial hotfix to enable it for a particular release.